code/src/components/FrequencyRadioOption.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+
+//onFrequencyChange will be the prop name in parent file "Form"
+export const FrequencyRadioOption = ({ frequencyValue, setFrequency }) => {
+
+    const onFrequencyChange = (e) => {
+        setFrequency(e.target.value)
+    }
```

Comment:
I like that the event handlers are in their respective components - makes it very readable 🥇